### PR TITLE
[ELY-2780] Logging in aggregaterealm changes authentication and authorization flow

### PR DESCRIPTION
**Pull request by lvydra**
https://issues.redhat.com/browse/ELY-2780

### CODE REVIEW

**Review by ivassile — APPROVED** [review]

**Comment by ivassile** [comment]
@fjuma When you have a chance, please review it. Thanks!

File: auth/realm/base/src/main/java/org/wildfly/security/auth/realm/AggregateSecurityRealm.java [hunk 1]
```diff
@@ -79,18 +79,15 @@ public AggregateSecurityRealm(final SecurityRealm authenticationRealm, Function<
     public RealmIdentity getRealmIdentity(final Evidence evidence) throws RealmUnavailableException {
         boolean ok = false;
         final RealmIdentity authenticationIdentity = authenticationRealm.getRealmIdentity(evidence);
-        if (authenticationIdentity.exists()) {
-            log.tracef("Authentication identity for principal [%s] found.", evidence.getDecodedPrincipal());
-        }
+        log.tracef("Authentication identity for principal [%s] found.", evidence.getDecodedPrincipal());
```
Comment (darranl) [inline auth/realm/base/src/main/java/org/wildfly/security/auth/realm/AggregateSecurityRealm.java]: I think we may need to think about the message a bit more as it is now subtly different, we can confirm we obtained a RealmIdentity from the authenticationRealm but not sure we can fully declare it "found" as we no longer have a "not found" flow.

Comment (lvydra) [reply auth/realm/base/src/main/java/org/wildfly/security/auth/realm/AggregateSecurityRealm.java]: Hi @darranl, thank you for the review, I have updated the messages.

**Review by darranl — APPROVED** [review]

**Comment by darranl** [comment]
Thank you @lvydra

**Pull request merged**